BUILD

```diff
@@ -143,3 +105,70 @@ py_binary(
         requirement("yarl"),
     ],
 )
+
+#pkg_files(
+#    name = "python_files",
+#    srcs = ["@python39_x86_64-unknown-linux-gnu//:files"],
+#    strip_prefix = "",
+#)
+
+#pkg_tar(
+#    name = "python_layer_tar",
+#    srcs = [":python_files"],
+#)
+
+#container_layer(
+#    name = "python_layer",
+#    directory = "/usr/local",
+#    tars = [":python_layer_tar"],
+#)
+
+#container_image(
+#    name = "python_base",
+#    base = "@amazon2_linux//image",
+#    layers = [":python_layer"],
+#)
+
+container_image(
+    name = "python_compat",
+    base = "@python39_image//image",
+    symlinks = {
+        "/usr/bin/python": "/usr/local/bin/python"
+    }
+)
+
+py_layer(
+    name = "pip_tools_deps",
```

Comment:
This list would need to be maintained but it is useful in order to match the same pip-tools that is used for `:requirements_test`.